File: 268. Missing Number/solution.py
class Solution(object):
  def missingNumber(self, nums):
    """
    :type nums: List[int]
    :rtype: int
    """
    total = 0
    for index, num in enumerate(nums):
      total += num
      total -= index
    total -= len(nums)
    return abs(total)

File: 268. Missing Number/solution.cpp
class Solution {
public:
  int missingNumber(vector<int>& nums) {
    int total = 0;
    for (int index = 0; index < nums.size(); index++){
      int num = nums[index];
      total += num;
      total -= index;
    }
    total -= nums.size();
    return abs(total);
  }
};